docs/source/pipelines.rst

```diff
@@ -118,38 +119,106 @@ Key concepts
 
 Usage
 -----
+Development environments
+~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend to develop models with MLflow Pipelines with one of the following environment settings:
 
-The general workflow for using MLflow Pipelines is as follows:
+[**Databricks**]
+  - Edit YAML config and Python files in Databricks Repos. Open separate browser tabs for each
+    file module that you wish to modify.
+  - Use ``notebooks/databricks.py`` as the driver to run pipeline steps and inspect its output.
```

Comment:
It feels like these instructions are missing a few steps for project construction and git repo initialization. Should we first tell users to fork the template repo and/or pull the repo into Databricks Repos? Otherwise, we might give off the impression that we expect users to construct the YAML, python, and notebook files on their own. This also applies to the other environment settings.

Reply:
Oh, I see that cloning the template is mentioned below in the `Model development workflow` section. Maybe that section should be above the `Development environments` section?

Reply:
Agreed, we should start with how to clone the repo on Databricks and then talk about which tabs to open.

Reply:
Shall we also tell users to open the notebook in a separate tab and also explain that this `databricks.py` file is converted to a notebook when the repo is cloned with Databricks?

Reply:
Good point! Swapped the two sections. On explaining "explaining that this databricks.py file is converted to a notebook when the repo is cloned with Databricks", I'm not sure how to best phrase it. In addition, is it a useful information to let users know about this product detail? As long as they open `notebooks/databricks.py`, they should realize what's going on.